.github/workflows/conda_build_and_publish.yml

```diff
@@ -26,4 +26,6 @@ jobs:
         publish: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') }}
         test_all: ${{(github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')) || (github.ref == 'refs/heads/master')}}
         additional_apt_packages: 'libegl1-mesa libegl1-mesa-dev'
+        convert_win: 'true'
+        convert_osx: 'true'
```

Comment:
we don't want this, as the conversion made huge packages.

Reply:
what do we do then? I need a windows version